extruct/opengraph.py

```diff
@@ -0,0 +1,100 @@
+import re
+
+import lxml.html
+
+import functools
+from .pathfinder import find
+from .transform import clean_missing
+from .schemaorg import (
+    schemaorg,
+    schemaorg_nested,
+    schemaorg_instock,
+)
+
+
+def opengraph_availability(avail):
```

Comment:
While transformation to schema.org syntax is useful, I'm not sure extruct is a right place for it. I think it makes sense to keep only extraction in `extruct`, and have a separate package for transforming everything to the same data format.